proofs/beacon_block_proofs_bellatrix/beacon_block_proof-15539558-cdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01.yaml

```diff
@@ -0,0 +1,33 @@
+# EL block header: 0xcdf9ed89b0c43cda17398dc4da9cfc505e5ccd19f7c39e3b43474180f1051e01
```

Comment:
How are you getting the block hash in your tests? Do you get it from the filename?

Perhaps would be slightly better if it was also in the test vectors as either separate file or another object in the same file?

Reply:
I will add the block hash as another object in the YAML file.